### PR TITLE
Fishing map/improve range slider

### PR DESCRIPTION
**Pull request by satellitestudiodesign**
- [x] Format the number to fit the space
Screenshot 2022-03-18 at 11 09 25

- [x] Allow having only min and max in dataset
Screenshot 2022-03-18 at 11 10 20

- [x] Calculate step based on range and width of slider
- [x] Avoid sending min a max filters id unneeded

### CODE REVIEW

File: libs/ui-components/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,363 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (j8seangel) [inline libs/ui-components/yarn.lock:1]: why did you need to run `yarn install` in this lib folder? did you have any error? or can we remove this file?

File: libs/dataviews-client/src/resolve-dataviews.ts [hunk 2]
```diff
@@ -327,10 +327,21 @@ export function resolveDataviews(
           const datasetSchema = dataviewInstance.datasets?.find(
             (d) => d.schema?.[filterKey] !== undefined
           )?.schema?.[filterKey]
+
           if (datasetSchema && datasetSchema.type === 'number') {
-            return `${filterKey} >= ${filterValues[0]} AND ${filterKey} <= ${
-              filterValues[filterValues.length - 1]
-            }`
+            const minPossible = Number(datasetSchema.enum[0])
```
Comment (j8seangel) [inline libs/dataviews-client/src/resolve-dataviews.ts:332]: 👏

**Review by j8seangel — COMMENTED** [review]
LGTM

File: libs/ui-components/yarn.lock (hunk 1 above)
Comment (satellitestudiodesign) [reply libs/ui-components/yarn.lock:1]: I don't remember it, so just a mistake I suppose. I'll delete it
